pkgs/development/libraries/libportal/default.nix

```diff
@@ -55,6 +55,10 @@ stdenv.mkDerivation rec {
     "-Dintrospection=${if variant != "qt5" then "true" else "false"}"
   ];
 
+  postPatch = lib.optionalString stdenv.isDarwin ''
+    substituteInPlace libportal/portal.c --replace "sys/vfs.h" "sys/mount.h"
+  '';
```

Comment:
postPatch is normally but after src. Can we try to upstream this?

Reply:
In GNOME, we group postPatch with the other phases and hooks, see https://github.com/jtojnar/nixpkgs-hammering/blob/0c54b7121a21f79e580773a46fc8d51482dc1d1c/explanations/attribute-ordering.md. But +1 for upstreaming the change.

Reply:
Created PR here: flatpak/libportal#106.

Reply:
My intention was that we get a patch file we can use which won't silently break, so to finish this we should fetchpatch the commit from the PR.

Reply:
Added the patch as a file, as well as a link to the PR in the comment.